source/Calamari.Shared/Integration/Packages/NuGet/NuGetV3LibDownloader.cs

```diff
@@ -25,31 +25,27 @@ public static void DownloadPackage(string packageId, IVersion version, Uri feedU
             if (feedCredentials != null)
             {
                 var cred = feedCredentials.GetCredential(feedUri, "basic");
-                sourceRepository.PackageSource.Credentials = new PackageSourceCredential("octopus", cred.UserName, cred.Password, true);
+                sourceRepository.PackageSource.Credentials = new PackageSourceCredential("octopus", cred.UserName, cred.Password, true, null);
```

Comment:
Based on comment in decompiled source for this class (cannot find link in GitHub, sorry), passing now treats all authentication types as valid, presumably treated the same as the missing value in the original signature to the right.

        /// <summary>
        /// Converts an authentication type filter string from the config file syntax to a list of valid authentication
        /// types
        /// </summary>
        /// <param name="str">
        /// Comma-delimited list of authentication types the credential is valid for as stored in the config file.
        /// If null or empty, all authentication types are valid. Example: 'basic,negotiate'
        /// </param>
        /// <returns>
        /// Enumeration of valid authentication types. If empty, all authentication types are valid.
        /// </returns>